convert_json_to_txt.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+import os
+import cv2
+import json
+
+# This script converts VIA (VGG image Annotator) json files to .txt files required for YOLOv3.
+
+def create_txt_files(via_json_file, images_folder, txt_dest_folder, attribute_name, labels):
+	"""
+	Input:
+	via_json_file = json files exported from VIA annotator
+	images_folder = path folder containing images
+	txt_dest_folder = destination folder for txt files
+	label_idx_dictionary = dictionary whose keys are label names
+						   in VIA annotator and values are the 
+						   corresponding integers to be use for 
+						   YOLOv3. For example: 
+						   {'label0': 0, 'label1':1, 'label3': 3}
+	attribute_name = attribute name assigned from VIA annotator
+	Output:
+		Side effects: it creates .txt files whose names correspond
+					to the images names in train. 
+		None
+	"""
+	label_idx_dictionary = labels 
+	print(attribute_name)
+	data = json.load(open(via_json_file, "r"))
+	for name in list(data.keys()):
```

Comment:
here you can do `for (name, file_data) in data.items()` (no need for `list()` either)